app/assets/stylesheets/trips.css.scss

```diff
@@ -58,22 +48,22 @@
   border: solid 3px #363636;
 }
 
-.call-to-action {
+.call-to-action, .call-to-action:visited, .call-to-action:link {
   display: block;
-  font-size: 1.2em;
+  font-size: 1.3em;
   color: white;
   background-color: #544e9c;
-  padding: 30px 0 30px 30px;
+  padding: 30px 0 30px 0px;
   text-decoration: none;
   border: solid 1px #363636;
-  border-radius: 5px 5px 5px 5px
+  border-radius: 5px 5px 5px 5px;
+  text-align: center;
 }
 
 .call-to-action:hover {
-  text-decoration: underline;
-  -moz-box-shadow:    0px 3px 5px 0px #aaa;
-  -webkit-box-shadow: 0px 3px 5px 0px #aaa;
-  box-shadow:         0px 3px 5px 0px #aaa;
+  -moz-box-shadow:    0px 2px 15px 0px rgba(0, 0, 0, 0.4);
```

Comment:
Colon after property should be followed by one space<br>`0px` should be written without units as `0`